src/pybind/mgr/volumes/module.py

```diff
@@ -63,7 +63,7 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'cmd': 'fs subvolume create '
                    'name=vol_name,type=CephString '
                    'name=sub_name,type=CephString '
-                   'name=size,type=CephInt,req=false '
+                   'name=size,type=CephString,req=false '
```

Comment:
This seems like the wrong direction to fix this. Why not fix `self.fs.setxattr(path, 'ceph.quota.max_bytes', size.encode('utf-8'), 0)` to convert the integer to a string and then to a byte array? Otherwise we lose the valuable type checking in the above code.

Reply:
Makes sense. Updated.